manual/src/cmds/intf-c.etex

```diff
@@ -946,10 +946,13 @@ void foo (value v1, value v2, value v3)
 }
 \end{verbatim}
 
-\paragraph{Note:} if your function is a primitive with more than 5 arguments
+\paragraph{Note:} If your function is a primitive with more than 5 arguments
 for use with the byte-code runtime, its arguments are not "value"s and
 must not be declared (they have types "value *" and "int").
 
+\paragraph{Warning:} If using "CAMLreturn0", it should only be used for internal FFI procedures of type void.
+If using "CAMLreturn(Val_unit)", it should be used for functions that return an OCaml unit vlaue.
```

Comment:
```suggestion
\paragraph{Warning:} "CAMLreturn0" should only be used for internal procedures that return void.
"CAMLreturn(Val_unit)" should be used for functions that return an OCaml unit value. Primitives (C functions that can be called from OCaml) should never return void.
```
Suggestions: simplify the structure of these two sentences. I also propose to avoid the use of the word “FFI” in favor of “primitive” for consistency with the rest of the chapter. My suggestion also contains an additional sentence to insist on the fact that functions called from OCaml should never return void.

Reply:
Thank you very much, Olivier! 😄 
Updated https://github.com/ocaml/ocaml/pull/11894/commits/78b817d3123f27abd95ebf2ca198576bc508fddb
Please help to review!